content/en/blog/falcosidekick-reponse-engine-part-3-knative.md

```diff
@@ -0,0 +1,473 @@
+kubectl apply -f https://github.com/knative/net-kourier/releases/download/v0.22.0/kourier.yaml
+kubectl patch configmap/config-network \
+  --namespace knative-serving \
+  --type merge \
+  --patch '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+
+# Installs Knative Eventing
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/eventing-crds.yaml
+kubectl wait --for=condition=Established --all crd
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/eventing-core.yaml
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/in-memory-channel.yaml
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/mt-channel-broker.yaml
+
+# Creates a default Broker
+kubectl create -f - <<EOF
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: default
+  namespace: default
+EOF
+```
+
+See also
+[knative.dev install instructions](https://knative.dev/docs/install/any-kubernetes-cluster/)
+for installing these into your own cluster.
+
+### Falco/Falcosidekick/sidekick UI
+
+We'll use helm to install `Falco` ,`Falcosidekick` and `Falcosidekick UI`.
+
+First, add the falcosecurity `helm` repo:
+
+```shell
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+```
+
+In a real project, you should get the whole chart with
+`helm pull falcosecurity/falco --untar` and then configure the `values.yaml`.
+For this tutorial, will try to keep thing as easy as possible and set configs
+directly by `helm install` command:
+
+```shell
+helm install falco falcosecurity/falco \
+  --create-namespace --namespace falco \
+  --set falcosidekick.enabled=true --set falcosidekick.webui.enabled=true \
+  --set falcosidekick.config.cloudevents.address=http://broker-ingress.knative-eventing.svc.cluster.local/default/default
+```
+
+You should get this output:
+
+```shell
+NAME: falco
+LAST DEPLOYED: Thu Jan 14 23:43:46 2021
+NAMESPACE: falco
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Falco agents are spinning up on each node in your cluster. After a few
+seconds, they are going to start monitoring your containers looking for
+security issues.
+No further action should be required.
+
+```
+
+And you can see your new `Falco`,`Falco Sidekick`,`Falco Sidekick UI` pods:
+
+```shell
+kubectl get pods -n falco
+NAME                                      READY   STATUS    RESTARTS   AGE
+falco-jh75c                               1/1     Running   0          1d
+falco-falcosidekick-554b8859d5-v9xkg      1/1     Running   0          1d
+falco-falcosidekick-554b8859d5-x2zkk      1/1     Running   0          1d
+falco-falcosidekick-ui-5d747688f9-g96x5   1/1     Running   11         1d
+```
+
+The arguments
+`--set falcosidekick.enabled=true --set falcosidekick.webui.enabled=true`
+enables Falcosidekick and the UI as the below shows:
+
+![](https://i.imgur.com/ZR25hRW.png)
```

Comment:
The UI has drastically changed with last release, can you update you screenshot please

Reply:
You can use this one https://raw.githubusercontent.com/falcosecurity/falcosidekick-ui/master/imgs/webui_02.png

And btw, you should add in assets of the website, no external that may disappear, please

Reply:
Done.